frontend/packages/console-shared/src/constants/index.ts

```diff
@@ -1 +1,8 @@
 export const DASH = '-';
+
+// Ceph based provisioners
+export const cephStorageProvisioners = [
```

Comment:
This should go in a file specific to storage

Reply:
Update ceph-storage-plugin to use this constant

Reply:
ack